python/lsst/base/packages.py

```diff
@@ -278,6 +327,7 @@ def fromSystem(cls):
         packages : `Packages`
         """
         packages = {}
+        packages.update(getCondaPackages())
```

Comment:
You mentioned a slowdown, does `fromSystem` need to be cached in some way? Or is generally processed once per process?

Reply:
We only expect this to be called once but on the other hand we can be pretty confident that it's not going to change whilst this process is running. It might be best to add some lru_cache around each of the get functions. @ktlim can you see a downside for doing that?

Reply:
Altough there is a special case of the routine that gets the python packages since that only reports packages that have been loaded so if you call it, then do an import, then call it again, you will get a different answer. That won't be a problem for the Conda stuff or, I imagine, the EUPS one.